generator/FORMAT.md

```diff
@@ -32,6 +32,8 @@ module_name:
      indexes:
       - labelname: ifIndex
         type: gauge
+        fixed_size: 0   # Only possible for OctetString/DisplayString.
```

Comment:
I would suggest making this a separate example, to make it more clear that this is for `type: DIsplayString`.  Also, comment that it's an optional field.

Reply:
Done